opendbc/car/ford/radar_interface.py

```diff
@@ -229,6 +245,67 @@ def _update_delphi_mrr(self, ret: structs.RadarData):
     if headerScanIndex != 3:
       return False
 
+    return self.do_clustering()
+
+  def _update_delphi_mrr_64(self, ret: structs.RadarData):
+    headerScanIndex = int(self.rcp.vl["MRR_Detection_001"]['CAN_SCAN_INDEX_2LSB_01_01'])
+
+    # In reverse, the radar continually sends the last messages. Mark this as invalid
+    if (self.prev_headerScanIndex + 1) % 4 != headerScanIndex:
+      self.radar_unavailable_cnt += 1
+    else:
+      self.radar_unavailable_cnt = 0
+    self.prev_headerScanIndex = headerScanIndex
+
+    if self.radar_unavailable_cnt >= 5:
+      self.pts.clear()
+      self.points.clear()
+      self.clusters.clear()
+      ret.errors.radarUnavailableTemporary = True
+      return True
+
+    # Use points with Doppler coverage of +-60 m/s, reduces similar points
+    if headerScanIndex in (0, 1):
+      return False
+
+    for ii in range(1, DELPHI_MRR_RADAR_MSG_COUNT_64 + 1):
+      msg = self.rcp.vl[f"MRR_Detection_{ii:03d}"]
+
+      maxRangeID = 7 if ii < DELPHI_MRR_RADAR_MSG_COUNT_64 else 4 # all messages have 7 points except the last one, which has only 4 points in CANFD
```

Comment:
```suggestion
      maxRangeID = 7 if ii < DELPHI_MRR_RADAR_MSG_COUNT_64 else 4  # all messages have 7 points except the last one, which has only 4 points in CANFD
```

two spaces for comments for readability